src/types.ts

```diff
@@ -104,14 +104,13 @@ export type LastActiveTool =
 export type SidebarName = string;
 export type SidebarTabName = string;
 
-export type CommonCanvasAppState = {
+type _CommonCanvasAppState = {
   zoom: AppState["zoom"];
   scrollX: AppState["scrollX"];
   scrollY: AppState["scrollY"];
   width: AppState["width"];
   height: AppState["height"];
   viewModeEnabled: AppState["viewModeEnabled"];
-  editingElement: AppState["editingElement"];
   editingGroupId: AppState["editingGroupId"]; // TODO: move to interactive canvas if possible
   selectedElementIds: AppState["selectedElementIds"]; // TODO: move to interactive canvas if possible
   frameToHighlight: AppState["frameToHighlight"]; // TODO: move to interactive canvas if possible
```

Comment:
I've made the TODO comment because these three props conceptually did not seem like a reason for re-rendering the StaticCanvas - all of them seemed related only to the interactive part.

It looks like my assumption was incorrect here, but it still feels confusing and IMO should be still refactored away (not necessarily as part of this fix).